pkg/cmd/doc.go

```diff
@@ -0,0 +1,2 @@
+// Package cmd defines various public commands and internal commands (e.g. to serve the playground).
```

Comment:
SpiceDB does not serve the playground

Reply:
Right. But for example, the LSP command, isn't that used by the playground?

Reply:
I'm not aware the Playground has been updated to use the LSP. I know it was developed for the Visual Studio Code extension, and any IDE that supports LSP

Reply:
Ok, removed references to the playground.